helm/logging-operator/values.yaml

```diff
@@ -38,6 +38,7 @@ networkPolicy:
   flavor: cilium
 
 loggingOperator:
+  defaultWorkloadClusterNamespaces: "kube-system,giantswarm"
```

Comment:
Sure the name is fine but we will probably change it to defaultNamespaces or defaultStaticNamespaces later right?

Reply:
I though `defaultNamespaces` was confusing because this will only affect workload clusters.

Reply:
For now yes but not once https://github.com/giantswarm/roadmap/issues/3715 is done :)

Reply:
Renamed it to `defaultNamespaces`